### PR TITLE
Add wtsapi32.

### PR DESCRIPTION
**Pull request by ekse**
Based on 10.0.17134.0\um\WtsApi32.h.

### CODE REVIEW

File: src/um/wtsapi32.rs
Comment (Connicpu) [inline src/um/wtsapi32.rs]: Shouldn't `WTS_CURRENT_SERVER_NAME` be right here? ^^

File: src/um/wtsapi32.rs
Comment (Connicpu) [inline src/um/wtsapi32.rs]: `#define DOMAIN_LENGTH            17`

File: src/um/wtsapi32.rs
Comment (Connicpu) [inline src/um/wtsapi32.rs]: extra newline 😱 😂

File: src/um/wtsapi32.rs
Comment (Connicpu) [inline src/um/wtsapi32.rs]: These should be `[WCHAR; _]`

File: src/um/wtsapi32.rs
Comment (Connicpu) [inline src/um/wtsapi32.rs]: One parameter per line~

File: src/um/wtsapi32.rs
Comment (Connicpu) [inline src/um/wtsapi32.rs]: Again, one parameter per line~

File: src/um/wtsapi32.rs
Comment (Connicpu) [inline src/um/wtsapi32.rs]: Single parameter lines! 😛

File: src/um/wtsapi32.rs
Comment (Connicpu) [inline src/um/wtsapi32.rs]: SPL!

File: src/um/wtsapi32.rs
Comment (Connicpu) [inline src/um/wtsapi32.rs]: Alright, I'm done :P just split these into multiple lines!

**Review by Connicpu — CHANGES_REQUESTED** [review]
Mostly good, you've just got a few corrections to make! (mostly formatting, some omissions and incorrect bindings)

File: src/um/wtsapi32.rs
Comment (ekse) [reply src/um/wtsapi32.rs]: This one I didn't know what type to give it, it's defined as NULL, it's going to be either an LPSTR or a LPWSTR. Maybe for this one we could add WTS_CURRENT_SERVER_NAME_A and WTS_CURRENT_SERVER_NAME_W?

**Comment by ekse** [comment]
Thanks for the quick review, I'll do the fixes right away.

File: src/um/wtsapi32.rs
Comment (retep998) [reply src/um/wtsapi32.rs]: Just put it in as a comment for now so I can decide what to do with it later.

Comment (ekse) [reply src/um/wtsapi32.rs]: Done.

File: src/um/wtsapi32.rs
Comment (ekse) [reply src/um/wtsapi32.rs]: Fixed.

File: src/um/wtsapi32.rs
Comment (ekse) [reply src/um/wtsapi32.rs]: Fixed.

File: src/um/wtsapi32.rs
Comment (ekse) [reply src/um/wtsapi32.rs]: Fixed.

File: src/um/wtsapi32.rs
Comment (ekse) [reply src/um/wtsapi32.rs]: Fixed.

File: src/um/wtsapi32.rs
Comment (ekse) [reply src/um/wtsapi32.rs]: Fixed.

File: src/um/wtsapi32.rs
Comment (ekse) [reply src/um/wtsapi32.rs]: Fixed.

File: src/um/wtsapi32.rs
Comment (ekse) [reply src/um/wtsapi32.rs]: Fixed.

**Comment by ekse** [comment]
Anything still blocking this pull request?

**Comment by retep998** [comment]
Mostly just me finding the time to do a final review before merging.

File: src/um/wtsapi32.rs
Comment (retep998) [inline src/um/wtsapi32.rs]: The `isize` literal suffix shouldn't be needed here since it isn't negative and it is being cast to a pointer.

File: src/um/wtsapi32.rs
Comment (retep998) [inline src/um/wtsapi32.rs]: Why is the `{` on a new line?

File: src/um/wtsapi32.rs
Comment (retep998) [inline src/um/wtsapi32.rs]: Why is this indented weird?

**Review by retep998 — CHANGES_REQUESTED** [review]

File: src/um/wtsapi32.rs
Comment (ekse) [reply src/um/wtsapi32.rs]: removed

File: src/um/wtsapi32.rs
Comment (ekse) [reply src/um/wtsapi32.rs]: fixed

File: src/um/wtsapi32.rs
Comment (ekse) [reply src/um/wtsapi32.rs]: fixed

**Comment by svartalf** [comment]
Is there any reason this PR was closed unmerged?

**Pull request closed without merging**
